lib/streamlit/elements/layouts.py

```diff
@@ -370,12 +367,6 @@ def dg(self) -> "DeltaGenerator":
         return cast("DeltaGenerator", self)
 
     # Deprecated beta_ functions
-    beta_container = function_prerelease_graduation_warning(
-        PrereleaseAPIType.BETA, container, "2021-11-02"
-    )
-    beta_expander = function_prerelease_graduation_warning(
-        PrereleaseAPIType.BETA, expander, "2021-11-02"
-    )
-    beta_columns = function_prerelease_graduation_warning(
-        PrereleaseAPIType.BETA, columns, "2021-11-02"
-    )
+    beta_container = deprecate_func_name(container, "beta_container", "2021-11-02")
```

Comment:
should we remove from the codebase this `beta_` names at some point?

Reply:
We should!